juicy-select.html

```diff
@@ -51,6 +51,9 @@
                     }
                 }.bind(this), 10);
             },
+            connectedCallback : function() {
+                this.attached();
+            },
```

Comment:
Is this really needed? Polymer 2.0 is supposed to alias `attached` as `connectedCallback` in hybrid mode.